features/pull_request.feature

```diff
@@ -812,3 +812,36 @@ BODY
       Error creating pull request: Temporary Redirect (HTTP 307)
       Refused to follow redirect to https://disney.com/mouse\n
       """
+
+  Scenario: Default message with --push
+    Given the text editor adds:
+      """
+      """
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        halt 400 if request.content_charset != 'utf-8'
+        assert :title => 'This is somewhat of a longish title that does not get wrapped & references #1234',
+               :body => nil
+        status 201
+        json :html_url => "the://url"
+      }
+      """
+    Given I am on the "master" branch pushed to "origin/master"
+    When I successfully run `git checkout --quiet -b topic`
+    Given I make a commit with message "This is somewhat of a longish title that does not get wrapped & references #1234"
```

Comment:
Your scenario is not meant to assert that hub successfully uses the message of a single commit for the pull request title & body. Other scenarios are taking care of that already. You can just make an anonymous commit with `Given I make a commit`.